receiver/xreceiver/metadata.yaml

```diff
@@ -0,0 +1,9 @@
+type: profile
```

Comment:
While this type is correct today, `xreceiver` will eventually also have the entities implementation. Maybe `type: experimental`?

Reply:
is this something to change for all of the `x*` components?

Reply:
Yes!

Reply:
i updated this again. all the x* components now have type: <the component name> and class: receiver|processor|...
I think that is more appropriate and more in line with the other components